### PR TITLE
Tickets/dm 26233

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: ess/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+RUN git clone https://github.com/lsst-ts/ts_ess.git \
+  && git clone https://github.com/lsst-ts/ts_config_ocs.git
+RUN echo 'export TS_CONFIG_OCS_DIR=/home/ess/ts_config_ocs' >> .bashrc \
+ && echo 'export OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/armv7l.linux-debug' >> .bashrc \
+ && echo 'export OSPL_URI=file://${OSPL_HOME}/etc/config/ospl.xml' >> .bashrc \
+ && echo 'export PATH=${OSPL_HOME}/bin:${PATH}' >> .bashrc \
+ && echo 'export LD_LIBRARY_PATH=${OSPL_HOME}/lib' >> .bashrc \
+ && echo 'export LSST_DDS_DOMAIN=ess' >> .bashrc \
+ && echo 'export PYTHONPATH=/home/ess/ts_ess/python' >> .bashrc
+ENTRYPOINT /bin/bash
```
Comment (tribeiro) [inline ess/Dockerfile:48]: I don't think you need this `ENTRYPOINT` step.

File: generic_camera/canon/Dockerfile.aarch64 [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  && sed -i  's/default: Simulator/default: Canon/' ts_GenericCamera/schema/GenericCamera.yaml \
+  && chmod 755 ts_GenericCamera/bin.src/run_genericcamera.py
+RUN echo 'export TS_CONFIG_OCS_DIR=/home/allsky/ts_config_ocs' >> .bashrc \
+ && echo 'export OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/aarch64.linux' >> .bashrc \
+ && echo 'export OSPL_URI=file://${OSPL_HOME}/etc/config/ospl.xml' >> .bashrc \
+ && echo 'export PATH=${OSPL_HOME}/bin:${PATH}' >> .bashrc \
+ && echo 'export LD_LIBRARY_PATH=${OSPL_HOME}/lib' >> .bashrc \
+ && echo 'export LSST_DDS_DOMAIN=AllSky' >> .bashrc \
+ && echo 'export PYTHONPATH=/home/allsky/ts_GenericCamera/python' >> .bashrc
+ENTRYPOINT /bin/bash
```
Comment (tribeiro) [inline generic_camera/canon/Dockerfile.aarch64]: Not sure you need this last step.

File: generic_camera/canon/Dockerfile.armv7l [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  && sed -i  's/default: Simulator/default: Canon/' ts_GenericCamera/schema/GenericCamera.yaml \
+  && chmod 755 ts_GenericCamera/bin.src/run_genericcamera.py
+RUN echo 'export TS_CONFIG_OCS_DIR=/home/allsky/ts_config_ocs' >> .bashrc \
+ && echo 'export OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/armv7l.linux-debug' >> .bashrc \
+ && echo 'export OSPL_URI=file://${OSPL_HOME}/etc/config/ospl.xml' >> .bashrc \
+ && echo 'export PATH=${OSPL_HOME}/bin:${PATH}' >> .bashrc \
+ && echo 'export LD_LIBRARY_PATH=${OSPL_HOME}/lib' >> .bashrc \
+ && echo 'export LSST_DDS_DOMAIN=AllSky' >> .bashrc \
+ && echo 'export PYTHONPATH=/home/allsky/ts_GenericCamera/python' >> .bashrc
+ENTRYPOINT /bin/bash
```
Comment (tribeiro) [inline generic_camera/canon/Dockerfile.armv7l]: Not sure you need this last step

**Review by tribeiro — APPROVED** [review]
Are you pushing these images to Dockerhub? I think it would worth putting them there. If so, you may want to mention that in the `README` file.

File: ess/Dockerfile (hunk 1 above)
Comment (wvreeven) [reply ess/Dockerfile:48]: Thanks, I removed them from all files.

File: generic_camera/canon/Dockerfile.aarch64 (hunk 2 above)
Comment (wvreeven) [reply generic_camera/canon/Dockerfile.aarch64]: Removed.

File: generic_camera/canon/Dockerfile.armv7l (hunk 3 above)
Comment (wvreeven) [reply generic_camera/canon/Dockerfile.armv7l]: Removed.

**Comment by wvreeven** [comment]
Thanks for the suggestion to upload them to Dockerhub. I agree that that would be worth it. I'll contact Rob to give me permission to upload them and then I will do so. I'll upload the README accordingly when I do that and I'll open another, minor, ticket for that.

**Pull request merged**